pex/resolver.py

```diff
@@ -164,10 +164,52 @@ class Resolver(object):
 
   class Error(Exception): pass
 
+  @staticmethod
+  def _expand_and_maybe_adjust_platform(interpreter, platform=None):
+    # Adjusts `platform` if it is 'current' and does not match the given `interpreter` platform.
+    cur_plat = Platform.current()
+
+    given_platform = Platform.create(platform or 'current')
+    if cur_plat.platform != given_platform.platform:
+      # IE: Say we're on OSX and platform was 'linux-x86_64' or 'linux_x86_64-cp-27-cp27mu'.
+      return given_platform
+
+    ii = interpreter.identity
```

Comment:
The brevity is fine right here, but by line 186 I had to look back and ask what is `ii` again? 

Is the intent of `ii` to make things shorter, or is there also a cost to accessing `interpreter.identity` that you're trying to cache to avoid recomputing every time? If only the former, then I think it would be more readable to use `interpreter.identity` everywhere, such as `interpreter.identity.abbr_impl`.

Reply:
This is a bit of the style bit mentioned above but I've fixed. For context, Pex was written by @wickman and later maintained by @kwlzn and the predominant style is one they share which is favoring brevity in local scopes where you have the handful of lines right there for all the context you need.